### PR TITLE
Decompose ScaledTanh into ONNX operators

### PR DESCRIPTION
**Pull request by vinitra**
In preparation for experimental op deprecation in the next ONNX release (v1.5, opset 10). Last operator in #259.

### CODE REVIEW

File: onnxmltools/convert/common/_apply_operation.py [hunk 1]
```diff
@@ -420,6 +420,39 @@ def apply_parametric_softplus(scope, input_name, output_name, container, operato
...
         apply_mul(scope, [fName, aName], output_name, container)
 
+
+def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=None, alpha=[1.0], beta=[1.0]):
+
+    name = _create_name_or_use_existing_one(scope, 'ScaledTanh', operator_name)
+    if container.target_opset < 9:
+        if len(alpha) != 1 or len(beta) != 1:
+            raise ValueError('alpha and beta must be 1-element lists')
+        op_type = 'ScaledTanh'
```
Comment (wschin) [inline onnxmltools/convert/common/_apply_operation.py]: `op_type` is not necessary. We can just do `container.add_node('ScaledTanh', input_name, output_name, **attrs)

Comment (vinitra) [reply onnxmltools/convert/common/_apply_operation.py]: Thanks for the suggestion! Fixed in update.

File: onnxmltools/convert/common/_apply_operation.py [hunk 2]
```diff
@@ -420,6 +420,39 @@ def apply_parametric_softplus(scope, input_name, output_name, container, operato
         # g = a * f
         apply_mul(scope, [fName, aName], output_name, container)
 
+
+def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=None, alpha=[1.0], beta=[1.0]):
```
Comment (wschin) [inline onnxmltools/convert/common/_apply_operation.py]: Please don't do `alpha=[1.0]`. Use `None` instead. Pycharm can tell why.

Comment (vinitra) [reply onnxmltools/convert/common/_apply_operation.py]: Thanks for the catch!

**Pull request merged**